backend/cms/templates/settings/user.html

```diff
@@ -0,0 +1,37 @@
+{% extends "_base.html" %}
+{% load i18n %}
+
+{% block content %}
+{% load static %}
+
+
+<h2 class="heading font-normal mb-2">{% trans 'User settings' %}</h2>
+<h3>{% trans 'Registered 2FA key' %}</h3>
```

Comment:
I'm wondering if we should name this "Registered FIDO/FIDO2 2-factor-authentication keys". Opinions? But on the other hand this can be explained in the documentation.

Reply:
I added a description next to the table (a paragraph did not look good). Please have a look at the text :smile:

Reply:
Yeah I think this description is really good! :+1: